src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,71 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Parser;
+import duke.Storage;
+import duke.Ui;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.ToDo;
+
+/**
+ * Represents the command for the addition of new tasks to Duke's TaskList.
+ */
+public class AddCommand implements Command{
```

Comment:
I think there's a missing white space before the open curly brace here?

```suggestion
public class AddCommand implements Command {
```